src/components/page/Loading/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './style.css';
+
+/** interface for Ripple props */
+export interface RippleProps {
+  borderColor?: string;
+  borderStyle?: string;
+  borderWidth?: string;
+  height?: string;
+  minHeight?: string;
+  width?: string;
+}
+
+export const defaultProps: Partial<RippleProps> = {
```

Comment:
Looks like `Partial` is not necessary here.  Ama?

Reply:
For some reason eslint was complaining that the props need to be passed to the component where it has been used

Reply:
Does not sound like that should be fixed here, no?  I prefer if we try to stay close to the "[Parse don't validate](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/)" idea.  i.e. in this case let's not intentionally make our strong typing weaker when we don't need to.  However is this is a lot of work to debug right now I am fine if you create an issue.

Reply:
Cool i created an issue [here](https://github.com/OpenSRP/web/issues/28) so that we can address this later.